docs/source/quickstart.rst

```diff
@@ -167,6 +182,10 @@ To build a Bento, first create a :code:`bentofile.yaml` in your project director
 
 Next, use the :code:`bentoml build` CLI command in the same directory to build a bento.
 
+Please ensure that:
+1. The "service" name is the same as the python file name where you're service is declared
+2. That you are instantiating your load runners as part of the main path of execution in your python file, not inside of a function. This is because BentoML inspects the code to determine which models to save with the Bento.
```

Comment:
this is already enforced by the API design, since the service instance creation: `bentoml.Service(name, runners=[ .. ])` will need a list of runner instances.

Reply:
hmmm... yea I guess so... It seems a little weird to me that you have to instantiate the runner (even though the model isn't loaded) to go into something static like an annotation. But yea, I see what you're saying. Didn't know if this was a common python convention or not.

Reply:
@timliubentoml we should probably mention the runner is lazily loaded on the runner process?

Reply:
the current version I'm about to push completely removes this, and rather annotates the bentofile.yaml and makes a couple of these things slightly more clear. For #2, as you said, based on convention it would be difficult to misconfigure, so maybe leave that clarification out for now. Or add an extra comment on the code that does it to clarify